### DISCUSSION TITLE
Help with theming, how to remove translucency from Task Manager LXQtGroupPopup?

### DISCUSSION THREAD

**Discussion by n3ws** [lxqt/lxqt#2384] [Q&A] [answered]
Hi,

I'm a new user, impressive how snappy and lean LXQt is!

I'm on Debian testing (with LXQt 1.2.0), Openbox and Compton
with GLX rendering backend (that got rid of horizontal tearing in
xscreensaver).

I'm not really a fan of transparent menus (hard to read), so I tried to
disable them. I set 1.00 opacity everywhere in Window Effects, but
that did not help with the panel. So I started modifying the theme 
(debian theme). 

This section fixed the transparency for the Application Menu:
```
#MainMenu  QMenu {
    background-color: #285058;
    border: 1px solid #515048;
}
```

Changing the background alpha value from 50% to 100% made the 
Task Manager popup list more readable, but it is still partially transparent.
Maybe somebody knows how to get fully opaque menus? I tried many
approaches but I don't know qss well enough to get this fixed.

```
#TaskBar LXQtGroupPopup {
    background:rgba(0, 0, 0, 100%);
    border: 1px solid rgba(0, 0, 0, 40%);;
}
```
![image](https://user-images.githubusercontent.com/43077456/221401701-69ae5f8e-4faa-485a-a730-598c2a70ddcb.png)

**Comment by ghost** [comment]
You can modify the theme from Kvantum if its installed through Kvantum, if you dont need compton you can also remove it or change its config file..

**Comment by ghost** [comment, marked as answer]
Check out this link ?? --> https://forum.manjaro.org/t/disable-transparency-on-windows-and-application-menu/24532/4

**Reply by n3ws** [reply]
Thanks, it appears that the file to edit in Debian is ~/.config/compton.conf
There was a section where the opacity of tooltips was set to 0.7. Changed that
to 1.0 and that fixed the issue. It seems that LXQtGroupPopup is
technically a tooltip.


```
wintypes :
{
  tooltip :
  {
    fade = true;
    shadow = false;
    opacity = 1.0;
    focus = true;
  };
;

```
